.github/markets/pr_template.md

```diff
@@ -6,4 +6,7 @@ Each json file under the [configs](../../configs) folder correspond to their res
 |Field   |Type   |Required  |Description  |Notes   |
 |---|---|---|---|---|
 |`network`   |`string`   |true   |The network that the json file corresponds to  |The networks available are: **mainnet, testnet, devnet** |
-|`featured_markets`   |`string[]`   |true   |The array of market names which will be listed under the Featured tab on Demex's [Markets page](https://app.dem.exchange/markets)  |The market names listed here **MUST** match the market names listed under the Carbon [Markets API](https://api.carbon.network/carbon/market/v1/markets?pagination.limit=10000). |
\ No newline at end of file
+|`featured_markets`   |`string[]`   |true   |The array of market names which will be listed under the Featured tab on Demex's [Markets page](https://app.dem.exchange/markets)  |The market names listed here **MUST** match the market names listed under the Carbon [Markets API](https://api.carbon.network/carbon/market/v1/markets?pagination.limit=10000). |
+|`blacklisted_markets`   |`string[]`   |true   |The array of market names that are blacklisted.  |The market names listed here **MUST** match the market names listed under the Carbon [Markets API](https://api.carbon.network/carbon/market/v1/markets?pagination.limit=10000). |
```

Comment:
**Comment 1**
For the `Description` field of this row, can provide an explanation why markets will be blacklisted.
E.g.
_The array of market names that are blacklisted. A market can be blacklisted for a number of reasons, such as it being invalid/duplicate/wrongly-added/etc._

**Comment 2**
For the `Notes` field of this row, can also add a note that markets listed under `blacklisted_markets` cannot be under the `featured_markets` field at the same time.